### PR TITLE
Change to URLs

### PR DESCRIPTION
**Pull request by Phineas**
#166

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,1299 +76,1297 @@ Additionally, you can check out [#37](https://github.com/pirate/sites-using-clou
...
-- vultr.com ([no leaked data found in several search engine caches](https://github.com/pirate/sites-using-cloudflare/issues/130))
-- fastmail.com ([not affected](https://twitter.com/FastMail/status/834939787924557824), [#2](https://news.ycombinator.com/item?id=13720050))
-- 1password.com ([not affected](https://discussions.agilebits.com/discussion/comment/356869/#Comment_356869))
+- [authy.com](http://authy.com)
+- [coinbase.com](http://coinbase.com)
+- [bitcoin.de](http://bitcoin.de)
+- [betterment.com](http://betterment.com)
+- [transferwise.com](http://transferwise.com)
+- [prosper.com](http://prosper.com)
+- [digitalocean.com](https://digitalocean.com) ([no sensitive data found in multiple search engine caches](https://www.digitalocean.com/company/blog/digitalocean-your-data-and-the-cloudflare-vulnerability/))
```
Comment (ddymko) [inline README.md]: Why was this changed to "no sensitive data found in multiple search engine cashes" instead of "no leaked data found in several search engine caches" Just curious?

Comment (Phineas) [inline README.md]: Not sure, just had to fix that section because I accidentally removed that line due to my script messing up on certain parentheses

Comment (ddymko) [inline README.md]: ah! no worries then - could you change it back just to follow the guidelines we came up with in https://github.com/pirate/sites-using-cloudflare/issues/153?

Comment (Phineas) [inline README.md]: Done.
